src/cocotb_tools/runner.py

```diff
@@ -200,6 +200,14 @@ def _build_command(self) -> Sequence[_Command]:
     def _test_command(self) -> Sequence[_Command]:
         """Return command to run a test."""
 
+    def _use_external_viewer(self) -> bool:
+        """Return if an external viewer should be called after simulation when gui=True."""
```

Comment:
```suggestion
        """Return if an external viewer should be called after simulation when ``gui=True``."""
```

Reply:
Sure. I was a bit confused about this. What you suggest is what I personally like, but having written lots of documentation using NumpyDoc standard, it should probably be `*gui* is True` or something, but then gui is not an argument of this method...

I have now realized that you use the Google docstring standard, so will read up on that.